app/goal_routes.py

```diff
@@ -0,0 +1,148 @@
+from flask import Blueprint, jsonify, abort, make_response, request, current_app
+import requests
+from app import db
+
+from app.models.goal import Goal
+from .routes import get_task_from_db
+
+
+goal_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+def get_goal_from_db(goal_id):
+    try:
+        goal_id = int(goal_id)
+    except:
+        abort(make_response({'msg': f'Invalid id {goal_id}'}, 400))
+        
+    goal = Goal.query.get(goal_id)
+
+    return goal if goal else abort(make_response({'msg': f'No Goal with id: {goal_id}'}, 404))
+
+
+
+@goal_bp.route("", methods=['POST'])
+def create_goal():
+    request_body = request.get_json()
+    
+    if 'title' not in request_body:
+        return {"details": "Invalid data"}, 400
+    
+    new_goal = Goal(
+        title = request_body['title']
+    )
+    
+    db.session.add(new_goal)
+    db.session.commit()
```

Comment:
Great job using `db.session.<method-name>.` Session gives us access to the following:

* `db` is our app’s instance of SQLAlchemy.
* `session` represents our active database connection.
* By referencing db.session we can use SQLAlchemy’s methods to perform tasks like committing a change to a model, storing a new record of a model, and deleting a record.

By referencing `db.session.add()` you are able to use the SQLAlchemy method to store a new record of the Goal model. ⭐